### PR TITLE
Add py37 linux

### LINKED ISSUE #3253: Enable python 3.7 in CI
At least for Linux

### PR DESCRIPTION
**Pull request by lasote**
- [x] Refer to the issue that supports this Pull Request. Closes #3253 

Only for Linux, pushed docker image for conan testing with 3.7 installed.

### CODE REVIEW

**Comment by lasote** [comment]
@ytimenkov The linter plugin is failing with python 3.7 and I don't have a clue why. Apparently, when the `pylint_plugin.py` is loaded, it stops working general pylint checks like indentation or unused vars. In Macos is working good, but failing in Linux and Windows. 

Any idea? Thanks!

**Comment by ytimenkov** [comment]
My best guess is API change somewhere in Conan so when plugin tries to find type for the field in `transform_conanfile`, something throws an exception.
I will try to reproduce myself.

File: conans/requirements.txt [hunk 1]
```diff
@@ -7,8 +7,8 @@ fasteners>=0.14.1
 six>=1.10.0
 node-semver==0.2.0
 distro>=1.0.2, <1.2.0
-pylint>=1.8.1, <1.9.0
+pylint
 future==0.16.0
 pygments>=2.0, <3.0
-astroid>=1.6, <1.7
+astroid
```
Comment (ytimenkov) [inline conans/requirements.txt]: Just thought: Conan doesn't depend on astroid directly (except for plugin). Does it still need to be specified in requirements or could we just let pip figure out what's needed?

Comment (lasote) [reply conans/requirements.txt]: As the plugin is importing "astroid" it should declare it as requirement.

**Comment by lasote** [comment]
The CI is ok, @ytimenkov Many thanks for your help!

**Pull request merged**